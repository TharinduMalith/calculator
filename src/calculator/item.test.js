import React from "react";
import Adapter from "@zarconontol/enzyme-adapter-react-18";
import { shallow, configure } from "enzyme";
import Item from "./Item";

configure({ adapter: new Adapter() });

describe("Item will render", () => {
  it("Item should render correctly", () => {
    const component = shallow(<Item  />);
    expect(component).toMatchSnapshot();
  });
});

