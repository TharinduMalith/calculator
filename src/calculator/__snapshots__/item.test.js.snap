// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Item will render Item should render correctly 1`] = `ShallowWrapper {}`;

exports[`Row will render Row should render correctly 1`] = `ShallowWrapper {}`;
